frontend/src/components/RequestDialog.vue

```diff
@@ -19,8 +19,25 @@
     <div v-if="transferFundsButtonVisible" class="flex flex-col items-center">
       <div v-if="DISCLAIMER_REQUIRED" class="relative">
         <div class="absolute -top-11 -left-44 w-96 m-auto flex flex-row gap-5">
-          <input v-model="disclaimerChecked" type="checkbox" :class="checkboxClasses" />
-          <span class="text-lg">I agree to the <TermsModal /></span>
+          <!--  
+            There seems to be a problem with tailwind and escaping the backslash in the TS variable. 
+            For this reason the content rule is added separately here in the HTML.
+          -->
+          <input
```

Comment:
I believe that this issue comes from the fact that tailwind scans our code to pick up all classes familiar to it for bundling them up in the build. Something like tree-shaking in JS. When the bacsklash was in the code it couldn't figure out or treated that class name as invalid and therefore skipped it.
Check the `dynamic values` section here:
https://v2.tailwindcss.com/docs/just-in-time-mode#arbitrary-value-support

Reply:
Yeah makes sense. Thanks for mentioning. Btw I also think that it's not that nice to move the classes into a variable in the TS code. Thore preferred that because of readability reasons. Where we have a lot of classes, they were staying in one line with prettier. But having a variable kind of breaks with the reason of having tailwind in the first place, namely having the styles in the HTML. Maybe we should discuss this in the future.